swan-lake/integration-tutorials/content-based-message-routing.md

```diff
@@ -106,7 +106,7 @@ Follow the instructions given in this section to develop the service.
     }
     ```
 
-4. Generate record types corresponding to the payloads from the hospital backend services by providing samples of the expected JSON payloads.
+4. Generate record types corresponding to the payload from the hospital backend service by providing samples of the expected JSON payloads.
```

Comment:
Need to fix the rest of the sentence too. :)

Reply:
Oh My bad.